content/en/docs/spinnaker-user-guides/terraform-use-integration.md

```diff
@@ -6,16 +6,65 @@ aliases:
-
 At the core of the Terraform Integration is the Terraformer service. This service fetches your Terraform projects from source and executes various Terraform commands against them. When a `terraform` stage starts, Orca submits the task to Terraformer and monitors it until completion. Once a task is submitted, Terraformer fetches your target project, runs `terraform init` to initialize the project, and then runs your desired `action` (`plan` or `apply`). If the task is successful, the stage gets marked successful as well. If the task fails, the stage gets marked as a failure, and the pipeline stops.
 
-At a high level, a Terraform Integration stage performs the following actions when it runs:
+A Terraform Integration stage performs the following actions when it runs:
 
 1. Authenticates to your repo using basic authentication credentials you provide. This can be a GitHub token or a BitBucket username/password combination.
 2. Pulls a full directory from your Git repository.
 3. Optionally uses a Spinnaker artifact provider (Github, BitBucket, or HTTP) to pull in a `tfvars`-formatted variable file.
-4. Runs the Terraform action you select.   
+4. Runs the Terraform action you select.
+
+## Requirements
+
+Before you can use the Terraform Integration stage, verify that Armory's Terraform Integration for Spinnaker is enabled. Additionally, your Terraform code needs to be stored in either GitHub or BitBucket repo that the Armory Platform can access. You grant access as part of the enablement process.
+
+For more information, see [Enabling the Terraform Integration]({{< ref "terraform-enable-integration">}}).
+
+## Example Terraform Integration stage
+
+
+The following example describes a basic pipeline that performs the plan and apply Terraform actions. This is a simple example. For a more involved one, watch this [demo](https://youtu.be/_p8v-6_5DTI) of the Terraform Integration.
+
+The pipeline consists of three parts:
+
+**Plan stage**
+
+A Plan stage in a pipeline performs the same action as running the `terraform plan` command. Although a Plan stage is not strictly required since an Apply stage performs a `terraform plan` if no plan file exists, it's a good idea to have one. You'll understand why when during the Manual Judgment stage.
+
+you do not need to For this stage, configure the following:
+
+- For the **Terraform version**, pick any version, such as `0.13.3`.
+- For the **Action**, choose **Plan**.
+- **Main Terraform Artifact**
+  - Select **Expected Artifact > Define a new artifact** 
+    - Select **Account > gitrepo**
+    - In **URL**, add the URL to the Git repo that houses your Terraform code.
+- Under **Terraform Artifacts** 
```

Comment:
the section is "Produces Artifact" (it's all the way at the bottom instead of in the main section)

the flow would be Produces Artifact >> Define Artifact >> in Matches Artifact, select embedded-artifact